common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -315,7 +316,7 @@
   "feature.wynntils.miniMap.overlay.miniMap.followPlayerRotation.name": "Follow Player Rotation",
   "feature.wynntils.miniMap.overlay.miniMap.maskType.description": "Should the minimap be a rectangle or a circle? (Circle is unsupported as of now)?",
   "feature.wynntils.miniMap.overlay.miniMap.maskType.name": "Map Mask Type",
-  "feature.wynntils.miniMap.overlay.miniMap.name": "Mini Map Overlay",
+  "feature.wynntils.miniMap.overlay.miniMap.name": "Minimap",
   "feature.wynntils.miniMap.overlay.miniMap.poiScale.description": "How big should mini-map POIs be?",
```

Comment:
We are obviously not very consistent here. "Minimap", "Mini Map" and "Mini-map" are all used.

Should we standardize on "Minimap" as a single word? If so, the class should be "Minimap" and the i18n keys should be `.minimap`, not `.miniMap`.

I'm all for a standard, I don't care which one *), but we should be consistent.

*) Except "mini-map", that's just ugly.

Reply:
I agree

Reply:
Agree that we need a standard, or that it should be "Minimap" (and not "Mini Map"), or both of the above? :)

Reply:
"Minimap" should be standard in my opinion